Apps/Admin/Client/Components/Support/Covid19AssessmentSection.razor

```diff
@@ -0,0 +1,37 @@
+@using HealthGateway.Common.Data.Utils
+@inherits Fluxor.Blazor.Web.Components.FluxorComponent
+
+<MudText Class="mt-4" Typo="Typo.subtitle1">
+    Assessment History
+</MudText>
+
+<MudTable Class="mt-4"
+          Items="@Rows"
+          Loading="@IsLoading"
+          AllowUnsorted="false"
+          Breakpoint="@Breakpoint.Md"
+          HorizontalScrollbar="true"
+          Striped="true"
+          Dense="true"
+          data-testid="assessment-history-table">
+    <HeaderContent>
```

Comment:
I suggest adding a fixed width on the first column to keep the data closer together.  Date string lengths are fairly uniform.

```suggestion
    <ColGroup>
        <MudHidden Breakpoint="@Breakpoint.MdAndDown">
            <col style="width: 125px;" />
            <col />
        </MudHidden>
    </ColGroup>
    <HeaderContent>
```